rmgpy/kinetics/uncertainties.pyx

```diff
@@ -0,0 +1,89 @@
+#                                                                             #
+# Permission is hereby granted, free of charge, to any person obtaining a     #
+# copy of this software and associated documentation files (the 'Software'),  #
+# to deal in the Software without restriction, including without limitation   #
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,    #
+# and/or sell copies of the Software, and to permit persons to whom the       #
+# Software is furnished to do so, subject to the following conditions:        #
+#                                                                             #
+# The above copyright notice and this permission notice shall be included in  #
+# all copies or substantial portions of the Software.                         #
+#                                                                             #
+# THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR  #
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,    #
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE #
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER      #
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING     #
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER         #
+# DEALINGS IN THE SOFTWARE.                                                   #
+#                                                                             #
+###############################################################################
+import numpy as np
+from libc.math cimport exp, log, sqrt, log10
+from rmgpy.constants import R
+from rmgpy.quantity import Quantity
+import logging
+
+"""
+This module contains information related to kinetic uncertainties
+"""
+
+cdef class RateUncertainty(object):
+    """
+    Class for storing kinetic uncertainty information describes identically a normal distribution 
+    on Log(k) and a lognormal distribution on k by mu and var at a single temperature Tref also 
+    includes potentially useful uncertainty treatment variables
+    N = number of samples used to generate the distribution 
+    correlation = label identifying source of estimate for correlated error treatment
+    Note that correlated errors are expected to be associated only with mu (the bias of the distribution)
+    """
+
+    def __init__(self,mu,var,Tref,N=None,correlation=None):
+        self.Tref = Tref
+        self.correlation = correlation
+        self.mu = mu
+        self.var = var
+        self.N = N
+
+    def __repr__(self):
+        s = "RateUncertainty(mu={mu},var={var},Tref={Tref},".format(mu=self.mu,var=self.var,Tref=self.Tref)
+        if self.N is not None:
+          s += "N={0!r},".format(self.N)
+        if self.correlation is not None:
+          s += "correlation={0!r},".format(self.correlation)
```

Comment:
Fix indentation and add spaces after commas.

Reply:
fixed